aten/src/ATen/native/mps/operations/BitwiseOps.mm

```diff
@@ -262,12 +271,58 @@ void handle_tensor_scalar_binary_op(const at::Tensor& self, const at::Scalar& ot
  return _bitwise_op_out_mps(self, other, output, "xor");
 }
 
+at::Tensor& bitwise_not_out_mps (const at::Tensor& self, at::Tensor& output_) {
```

Comment:
Can we make this consistent with rest :
```
_bitwise_op_out_mps(self, other, output, "not");
```

Reply:
Alas, the same method can not be reused, as `not` is unary, rather than binary op. (But that's a good point, I should rename the template to be `_bitwise_binary_op_out_mps`

Reply:
Sounds good.